### PR TITLE
release: bump version to 11.0.0-rc.3 w/ changelog

### PR DESCRIPTION
**Pull request by annieyw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 11.0.0-rc.3 "merino-meadow" (2020-11-10)
+
```
Comment (crisbeto) [inline CHANGELOG.md:2]: Should we add a line of text saying that there are no changes between rc.2 and rc.3?

**Review by crisbeto — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (annieyw) [reply CHANGELOG.md:2]: Ok I will add that in

**Review by mmalerba — APPROVED** [review]
LGTM

**Pull request merged**